src/v/redpanda/application.cc

```diff
@@ -1458,6 +1461,33 @@ void application::wire_up_runtime_services(
           .get();
     }
 
+    syschecks::systemd_message("Starting datalake services").get();
+    construct_service(
+      _datalake_coordinator_fe,
+      node_id,
+      &raft_group_manager,
+      &partition_manager,
+      &controller->get_topics_frontend(),
+      &metadata_cache,
+      &controller->get_partition_leaders(),
+      &controller->get_shard_table())
+      .get();
+
+    construct_service(
+      _datalake_manager,
+      node_id,
+      &raft_group_manager,
+      &partition_manager,
+      &controller->get_topics_state(),
+      &controller->get_topics_frontend(),
+      &controller->get_partition_leaders(),
+      &controller->get_shard_table(),
+      &_datalake_coordinator_fe,
+      &_as,
+      sched_groups.datalake_sg(),
+      memory_groups().datalake_max_memory())
+      .get();
```

Comment:
Are these not flag guarded?

Reply:
ya not right now, I did it this way to avoid needing a cluster restart to enable datalake  but this definitely needs more discussion.

Reply:
Yeah, it's a tradeoff of avoiding risk vs convenience for beta testers.